src/scss/theme.scss

```diff
@@ -1,4 +1,186 @@
-@import './sphinx-bootstrap';
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400|Lato:400');
+
+html {
+  font-size: 15px;
+}
+
+body {
+  background-color: white;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  line-height: 1.65;
+  color: #333;
+  padding-top: 80px;
+}
+
+p {
+  margin-bottom: 1.15rem;
+
+  &.rubric {
+    border-bottom: 1px solid rgb(201, 201, 201);
+  }
+}
+
+a {
+  color: #005b81;
+  text-decoration: none;
+
+  &:hover {
+    color: #e32e00;
+    text-decoration: underline;
+  }
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  margin: 2.75rem 0 1.05rem;
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 400;
+  line-height: 1.15;
+}
+
+// offsetting html anchor titles to adjust for fixed header
+h2::before,
+h3::before,
+h4::before,
+h5::before,
+h6::before {
+  display: block;
+  content: '';
+  height: 80px;
+  margin: -80px 0 0;
+}
+
+h1 {
+  margin-top: 0;
+  font-size: 2.488em;
+  color: #130654;
+}
+
+h2 {
+  font-size: 2.074em;
+  color: #130654;
+}
+
+h3 {
+  font-size: 1.728em;
+}
+
+h4 {
+  font-size: 1.44em;
+}
+
+h5 {
+  font-size: 1.2em;
+}
+
+small,
+.text_small {
+  font-size: 0.833em;
+}
+
+hr {
+  border: 0;
+  border-top: 1px solid #e5e5e5;
+}
+
+/* -------------------------------------------------------------------------- */
+
+a.headerlink {
+  color: #c60f0f;
+  font-size: 0.8em;
+  padding: 0 4px 0 4px;
+  text-decoration: none;
+
+  &:hover {
+    background-color: #c60f0f;
+    color: white;
+  }
+}
+
+pre {
+  padding: 10px;
+  background-color: rgb(250, 250, 250);
+  color: #222;
+  line-height: 1.2em;
+  border: 1px solid rgb(201, 201, 201);
+  margin: 1.5em 0 1.5em 0;
+  box-shadow: 1px 1px 1px #d8d8d8;
+}
+
+.deprecated {
+  margin-bottom: 10px;
+  margin-top: 10px;
+  padding: 7px;
+  background-color: #ffe4e4;
+  border: 1px solid #f66;
+
+  p {
+    display: inline;
+  }
+}
+
+.topic {
+  background-color: #eee;
+}
+
+.seealso dd {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.viewcode-back {
+  font-family: Arial, sans-serif;
+}
+
+.viewcode-block:target {
+  background-color: #f4debf;
+  border-top: 1px solid #ac9;
+  border-bottom: 1px solid #ac9;
+}
+
+table.field-list {
+  border-collapse: separate;
+  border-spacing: 10px;
+  margin-left: 1px;
+
+  th.field-name {
+    padding: 1px 8px 1px 5px;
+    white-space: nowrap;
+    background-color: rgb(238, 238, 238);
+  }
+
+  td:first-child {
+    white-space: nowrap;
+  }
```

Comment:
Now this seems to be nested inside "table.field-list", while this was originally not the case?

Reply:
You can see the result of that at https://146-130237506-gh.circle-artifacts.com/0/html/demo/api.html#autosummary-table-and-api-stub-pages (compare with the table at https://pandas-sphinx-theme.readthedocs.io/en/latest/demo/api.html#autosummary-table-and-api-stub-pages). 

(that's also a reason to preserve the original comments, as that was trying to explain this is specifically for "autosummary tables")

Reply:
Ok, good remark. Deleted too eagerly.